cms/jinja2/templates/base_page.html

```diff
@@ -42,6 +42,15 @@
     {# Changes the location of the <main> tag from the base template - ensures it surrounds the header area #}
     {# This change should be adopted by the design system soon #}
     <main id="main-content">
+        {% if localised_version_not_available %}
+            <div class="ons-announcement">
+                <div class="ons-container">
+                    <div class="ons-panel ons-panel--announcement ons-panel--no-title">
+                        {{ _("This page is currently not available in your locale. The English version is shown instead.") }}
+                    </div>
+                </div>
+            </div>
```

Comment:
```suggestion
            {% from "components/panel/_macro.njk" import onsPanel %}
            {{-
                onsPanel({
                    "variant": "announcement",
                    "body": _("This page is currently not available in your locale. The English version is shown instead.")
                })
            -}}
```

Reply:
Addressed in [99db1b3](https://github.com/ONSdigital/dis-wagtail/pull/140/commits/99db1b3e24591fccea192c4a305f5199cb698de7)